### ISSUE TITLE
Different getUniqueID in different RN versions on Android

### ISSUE THREAD

**Issue by DenisDov** [react-native-community/react-native-device-info#862]
I have app in React Native 0.58.4 and installed `"react-native-device-info": "^0.26.4"`. When i use `DeviceInfo.getUniqueID()` in my Android 8 Oreo device i get result `8f80d1805723de81`. 

Now i decided to upgrade my project to React Native 0.61.2 and install `"react-native-device-info": "^5.0.1"`. So i init new project in new folder, replace App/ folder from old project, install libs, etc. I noticed that `getUniqueID()`method now changed to `getUniqueId()`. And now when i use `DeviceInfo.getUniqueId()` i have different id: `c09cfee05a0af6e0`.

I have tried to install `"react-native-device-info": "^0.26.4"` to new project, and changed options in `app/build.gradle` but none help:
```
    We have 3 options for deviceId:
    //Option 1 (latest):
    firebaseIidVersion = "19.0.1" // default: "19.0.1"
    //Option 2 (legacy GooglePlay dependency but using AndroidX):
    googlePlayServicesIidVersion = "17.0.0" // default: "17.0.0" - AndroidX
    //Option 3 (legacy GooglePlay dependency before AndroidX):
    googlePlayServicesIidVersion = "16.0.1" 
```
How to get my old id in the new version of React Native?

**Comment by mikehardy** [comment]
I don't think this is a device-info issue - I think this has more to do with how the unique ids work - examine the docs for the underlying ID and see if you fall in a case where it may have changed https://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID

**Comment by DenisDov** [comment]
Thanks, I read that link to the holes, nothing fits my case, I create new blank projects on RN 0.58.4 and RN 0.61.2, I run on the same Pixel 2 android 9.0 - different IDs. Maybe that problem in React Native.

**Comment by mikehardy** [comment]
Hmm - no it wouldn't be react-native, they don't touch the id - I think the only way you would get the same id is if you created a blank project with the old version, installed it, then (this is important) updated that project and installed the update without removing the previous version - I think if you remove a bundle id from a phone, it gets a new id on install. I'm going from memory though - I read the docs closely before but it's been a while

**Comment by DenisDov** [comment]
TL;DR: after fresh init in RN 0.61.2 folder `android/app` now contain file `debug.keystore`, which is not presented in old RN 058.2. So i replaced `debug.keystore` with `release.keysore`. And now i get right Android ID.

**Comment by mikehardy** [comment]
Ok - so it doesn't seem to be a bug then - more a very subtle interaction between parts / project setup. If there's anything specific we could put in the docs that could help I'd be open to a PR, but it sounds like as long as your keystore is the same (i.e., you sign the app the same) then you'll maintain ID continuity?

**Comment by DenisDov** [comment]
Not worth it, closed.
